### PR TITLE
use py3 style next

### PR DESCRIPTION
**Pull request by nickpell**
@dimagi/py3

### CODE REVIEW

File: corehq/apps/case_importer/util.py [hunk 1]
```diff
@@ -110,14 +110,14 @@ def from_workbook(cls, workbook):
     def get_header_columns(self):
         if self.max_row > 0:
             # remove None columns the library sometimes returns
-            return filter(None, self.iter_rows().next())
+            return filter(None, next(self.iter_rows()))
```
[the remark this answers is not in the document]
Comment (nickpell) [reply corehq/apps/case_importer/util.py]: ignoring pre-existing ```filter``` for now

**Comment by emord** [comment]
How confident are you with this? I think these will only work if each of the classes have __next__ defined right? Are you sure that these are classes with that?

**Comment by nickpell** [comment]
Each should already have ```.next()``` defined because it's already being called.  ```next()``` will call ```.next()``` according to [this](https://docs.python.org/2/library/functions.html#next).

**Comment by emord** [comment]
Agreed, but that won't work in python 3 https://docs.python.org/3.6/library/functions.html#next

I think to work in py3 there needs to be an alias `next = __next__` on the class.

**Comment by proteusvacuum** [comment]
I think `six` has a solution for this: https://pythonhosted.org/six/#six.next

**Comment by emord** [comment]
I don't think `six` is required for this python 2 has handled this since python 2.6.

My worry was that if a class is defined like:

```
class Class(object):
    def next():
        ...
```

That would work in python 2 and here, but not in python3 since python 3 needs `__next__` definied. IIRC you need to explictly declare a `next` and `__next__` method  like this: https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/dimagi/utils/csv.py#L26

It's not a problem if these are python iterators. Only if we've defined the next method on whatever these are iterating over

**Comment by nickpell** [comment]
@emord I went through these, and it looks like they are all python iterators, so these changes should be ok.

**Pull request merged**